tests/network/localnet/test_default_bridge.py

```diff
@@ -11,3 +12,17 @@
 def test_connectivity_over_migration_between_localnet_vms(localnet_server, localnet_client):
     migrate_vm_and_verify(vm=localnet_client.vm)
     assert is_tcp_connection(server=localnet_server, client=localnet_client)
+
+
+@pytest.mark.ipv4
+@pytest.mark.single_nic
+@pytest.mark.usefixtures("nncp_localnet")
+@pytest.mark.polarion("CNV-11925")
+def test_connectivity_post_migration_between_localnet_vms(migrated_localnet_vm, vms_localnet):
+    base_localnet_vm = vms_localnet[0] if vms_localnet[1] == migrated_localnet_vm else vms_localnet[1]
+
+    with client_server_active_connection(client_vm=base_localnet_vm, server_vm=migrated_localnet_vm, port=8888) as (
```

Comment:
I prefer to have the port defined as a constant for 2 reasons:
1. It is easier to track when you see the different ports one next to the other, in a single location.
2. It is easier to change or re-use later, if needed.

Reply:
> * It is easier to track when you see the different ports one next to the other, in a single location.

The port has a meaning only here... i.e. it has no context beyond the body of this test.
We can use the "default", but it adds no value, on the contrary... this is a new port and make a more interesting verification.

> * It is easier to change or re-use later, if needed.

I do not want to change it later. It is per this usage only. I.e. other tests can use something else.

Reply:
I think we better add a docstring that explains why we chose a different port and why this 8888 port in specific (defining that these 4 numbers are meaningless is also important for futures readers).
Suggestion:
    """
    A distinct port (8888) is used for this connection to verify
    the migrated VM's capability to host a new service on an arbitrary,
    available port. This differs from other standard ports used in the
    test setup (e.g., _IPERF_SERVER_PORT) to ensure a more robust
    validation of post-migration network functionality. The specific
    numeric value 8888 is not significant beyond this test's scope.
    """

Reply:
I understand your point. I still think that adding a short explanation as to why this port number was chosen and what is the meaning of it can be beneficial for future readers, but I won't insist.
You can decide on this, and mark this comment as resolved. Thank you.